doc/sphinxman/source/dft.rst

```diff
@@ -717,3 +717,100 @@ The "best-practice" input file for KS-DFT is::
     }
 
     energy('b3lyp')
+
+
+Advanced Functional Use and Manipulation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Using the ``dft_functional`` option in the energy procedure call, one is able to use the orbitals generated by DFT for correlated wavefunction methods::
```

Comment:
I'd say "keyword argument" rather than "option" so as to reserve the latter for things in read_options.cc

Reply:
Also please add a warning that if you tweak calculations (esp. MP2+PBE0) that only generic psivars will be defined so as to avoid contamination.